receiver/prometheusreceiver/testdata/config_k8s.yaml

```diff
@@ -0,0 +1,40 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: apps
+        kubernetes_sd_configs:
+        - role: pod
+          selectors:
```

Comment:
This particular feature wasn't supported until update to Prometheus v2.19.2 (#1301), so I added it to cover with test